packages/shell-chrome/src/utils.js

```diff
@@ -8,6 +8,16 @@ export function flattenData(data) {
   return flattenedData;
 }
 
+function mapDataTypeToInputType(dataType) {
+    switch(dataType) {
+        case 'boolean': return 'checkbox';
+        case 'number': return 'number';
+        case 'string':
```

Comment:
Redundant case here, will fall through to default anyway.

Reply:
Yeah was just being explicit, I'll switch to having a comment there instead